ghcide/src/Development/IDE/Core/UseStale.hs

```diff
@@ -82,8 +82,8 @@ dualPositionMap (PositionMap (P.PositionMapping (P.PositionDelta from to))) =
 -- a 'PositionMapping' that will fast-forward values to the current age.
 data TrackedStale a where
   TrackedStale
-      :: Tracked (Stale s) a
-      -> PositionMap (Stale s) Current
+      :: Tracked ('Stale s) a
```

Comment:
Can we just not include this warning? I think `-Wno-unticked-promoted-whatever` is now generally considered to be a bad idea.

Reply:
I agree, unticked is the way to go nowadays.